src/PantheonTerminalWindow.vala

```diff
@@ -319,6 +319,17 @@ namespace PantheonTerminal {
             });
         }
 
+        public bool handle_paste_event () {
```

Comment:
Could this be called also here?
https://github.com/LightSunrider/terminal/blob/97fbeeb5949e907b988fc0f20534a1fd41d62180/src/PantheonTerminalWindow.vala#L309

This is the same code as in this method.